src/win/thread.c

```diff
@@ -38,6 +38,8 @@ typedef struct {
   uv__once_cb callback;
 } uv__once_data_t;
 
+#define MAX_THREAD_NAME_LENGTH 32767
```

Comment:
Is this needed? It doesn't appear to be used in this file and has the same value as `UV_PTHREAD_MAX_NAMELEN_NP`.

Reply:
Thanks for pointing this out. I forgot to remove.